packages/analytics-browser/src/det-notification.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  isSessionTrackingEnabled,
+  isFileDownloadTrackingEnabled,
+  isFormInteractionTrackingEnabled,
+  isPageViewTrackingEnabled,
+} from '@amplitude/analytics-client-common';
+import { BrowserConfig, LogLevel } from '@amplitude/analytics-types';
+
+let notified = false;
+
+export const detNotify = (config: BrowserConfig): void => {
+  if (notified) {
+    return;
+  }
+
+  const enabledTracking = [
+    isPageViewTrackingEnabled(config.defaultTracking) ? 'Page Views' : '',
+    isSessionTrackingEnabled(config.defaultTracking) ? 'Sessions' : '',
+    isFileDownloadTrackingEnabled(config.defaultTracking) ? 'File Downloads' : '',
+    isFormInteractionTrackingEnabled(config.defaultTracking) ? 'Form Interactions' : '',
+  ].filter(Boolean);
+  const enabledTrackingString = enabledTracking.join(', ');
+
+  if (enabledTracking.length) {
+    const message = `Your Amplitude instance is configured to track ${enabledTrackingString}. Visit https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events for more details.`;
+    config.loggerProvider.log(message);
+    /* istanbul ignore if */
+    if (config.logLevel < LogLevel.Verbose) {
+      console.log(message);
+    }
```

Comment:
i added a duplicate log here directly to console since default log levels would only log errors and warnings

Reply:
@yuhao900914 i added this after you reviewed, so tagging you in case you have feedback also

Reply:
LGTM.